external/vexflow/vexflow.d.ts

```diff
@@ -321,12 +321,19 @@ declare namespace Vex {
 
         export class CanvasContext extends RenderContext {
             public vexFlowCanvasContext: CanvasRenderingContext2D;
+            // TODO CanvasRenderingContext2D isn't defined anywhere, and defining it leads to problems.
+            // this somehow works, though.
+            public background_fillStyle: { fill: string };
+            public setBackgroundFillStyle(style: string): void;
+            public clear(x: number, y: number, width: number, height: number): void;
         }
```

Comment:
I really don't like the idea that I need to know my canvas size to clear it.

Reply:
perhaps you need to know it to fill it with the background color though. I know in any case that canvas.width and canvas.height is often wrong in the canvas backend, too small.